servlet/pom.xml

```diff
@@ -29,10 +29,9 @@
     </parent>
 
     <groupId>io.undertow</groupId>
-    <artifactId>undertow-servlet</artifactId>
+    <artifactId>undertow-servlet-jakarta</artifactId>
```

Comment:
I am not sure it is a good idea to change the module name. I thought we were just going to update the module, and from 2.3.0.Final forward, undertow-servlet uses Jakarta.